sdk/mobile-center-updates/src/main/java/com/microsoft/azure/mobile/updates/Updates.java

```diff
@@ -509,6 +509,17 @@ private synchronized void resumeUpdateWorkflow() {
                 return;
             }
 
+            /*
+             * If network is disconnected, browser will fail so wait.
+             * Also we can't just wait for network to be up and launch browser at that time
+             * as it's unpredictable and will interrupt the user, so just wait next relaunch.
+             */
+            if (!NetworkStateHelper.getSharedInstance(mContext).isNetworkConnected()) {
+                MobileCenterLog.info(LOG_TAG, "Postpone enabling in app updates via browser as network is disconnected.");
```

Comment:
It is a little bit confusing *enabling* in log. You are postponing a process of getting update token.